### ISSUE TITLE
Print from sd-viewer

### ISSUE THREAD

**Issue by deeplow** [freedomofpress/securedrop-workstation#1137]
## Description

Some may find confusing the fact that sd-viewer has a print menu, but doesn't actually allow you to print (https://github.com/freedomofpress/securedrop-workstation/issues/278). What it we made it work?

(**Update:** Nov 2025) After some digging it turns out this had already been considered in the past, so I guess it was a re-discovery. Here are some of those comments:

> I don't feel it's a bad experience to require users to open something before printing it, and in many ways that may be a more desirable flow. Especially since LibreOffice has solved so many other design problems, such as queuing and the "Print Options" dialog. I do expect users will likely wish to queue multiple print jobs, and the ability to manage that would be highly desirable.
> 
> — _Originally posted by @ninavizz in [#267](https://github.com/freedomofpress/securedrop-workstation/issues/267#issuecomment-508242927)_

## How will this impact [SecureDrop/SecureDrop Workstation users](https://github.com/freedomofpress/securedrop-ux/wiki/Users)?

Improved usability since users could print right away the documents they find worth printing.

## How would this affect the SecureDrop Workstation [threat model](https://github.com/freedomofpress/securedrop-workstation/#threat-model)?

<strike>If implemented properly (i.e. unidirection communication from sd-viewer to the print vm), it shouldn't add any more risks.</strike>
This adds the risk that a malicious document could export / print itself. Since a malicious file could potentially compromize sd-viewer, it follows that it could order its printing.

## User Stories

As as journalist I would like to be able to print documents immediately.

**Comment by rocodes** [comment]
See https://github.com/freedomofpress/securedrop-workstation/issues/278 https://github.com/freedomofpress/securedrop-workstation/issues/564

**Comment by rocodes** [comment]
I agree it would be great to print from sd-viewer and/or to have sd-viewer not be a "dead end".  Somewhere (maybe in the "print architecture" ticket, or maybe in its own epic) we should figure out a better overall data flow that allows material to flow through the VMs, whether that's a sanitized print rpc that we've talked about occasionally upstream, or an 'export -> dz/sanitize -> print' workflow, or the "briefcase" / one-disp-per-source idea that you had a few years back @deeplow, or what  :)
